lib/fastly/client.rb

```diff
@@ -14,14 +13,11 @@ def initialize(opts)
       @password = opts.fetch(:password, nil)
       @customer = opts.fetch(:customer, nil)
 
-      certificate_store = opts.fetch(:certificate_store, ROOT_CA)
-
       base      = opts.fetch(:base_url, 'https://api.fastly.com')
       uri       = URI.parse(base)
       ssl       = uri.is_a? URI::HTTPS  # detect if we should pass `use_ssl`
       @http     = Net::HTTP.start(uri.host, uri.port, use_ssl: ssl)
 
-      http.ca_path      = certificate_store
       http.verify_mode  = OpenSSL::SSL::VERIFY_PEER
```

Comment:
As far as I can see, it's [already done when use_ssl is set to true](https://github.com/ruby/ruby/blob/60c706177077f97c8feedddfa36c352205b0728b/lib/net/http.rb#L575-L576).